image/exposure_histogram/impl.py

```diff
@@ -0,0 +1,127 @@
+# Python built-ins, required for purely nicety reasons like raising warnings
```

Comment:
Block should also include authorship, license, copyright, dates and description of what the driver does.